app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java

```diff
@@ -1,23 +1,94 @@
 package ar.com.wolox.android.example.ui.example.login;
 
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
 import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.example.ExampleActivity;
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
 
 /**
  * A simple {@link WolmoFragment} subclass.
  */
 public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
 
+    @BindView(R.id.vLoginEmailEditText)
```

Comment:
Try to write BindViews, BindColors, BindDimens, etc. in the same line, such as the following:
@BindView(R.id.vLoginEmailEditText) EditText vLoginEmailEditText

Reply:
✔ **Done**